po/es.po

```diff
@@ -1663,19 +1663,19 @@ msgstr "Capacidades"
 
 #: modules/devices/arm/processor.c:335
 msgid "SOC/Package"
-msgstr ""
+msgstr "SOC/Paquete"
 
 #: modules/devices/arm/processor.c:338 modules/devices/cpu_util.c:222
 msgid "Topology"
 msgstr "Topologia"
 
 #: modules/devices/arm/processor.c:339
 msgid "Clocks"
-msgstr ""
+msgstr "Ciclos"
 
 #: modules/devices/arm/processor.c:354
 msgid "SOC/Package Information"
-msgstr ""
+msgstr "Informacion SOC/Paquete"
```

Comment:
yeah, there's a missing accent here, i revised and seems few only few accent are missing..  not so much..  also a tip: when i revised the rusian hardinfo, i could see that XFce4 are very bad translated.. seems some strings are not same in context.

all my pulls are editable by owers and collaborators, so if you or bp0 prefer can also edited and commited to ... 

so for what i mention: i can sure that my es.po are all in context and spell-related corrected!